font-codegen/src/parsing.rs

```diff
@@ -1662,6 +1666,9 @@ impl Count {
                 (CountTransform::SubAddTwo, [a, b]) => {
                     quote!(transforms::subtract_add_two(#a, #b))
                 }
+                (CountTransform::Custom, [a]) => {
+                    quote!(transforms::custom_count(#a))
```

Comment:
I'd probably just generate the code directly here, like:

```suggestion
                    quote!(usize::try_from(#a).unwrap_or_default())
```